app/src/main/java/com/kickstarter/viewmodels/BackingAddOnsFragmentViewModel.kt

```diff
@@ -24,6 +24,7 @@ import com.kickstarter.ui.fragments.BackingAddOnsFragment
 import rx.Observable
 import rx.subjects.BehaviorSubject
 import rx.subjects.PublishSubject
+import java.util.*
```

Comment:
nit: Why was this import added? Is it possible to not use the wild card and instead list each import individually? I just think it's easier for readability.